core/src/components/item/test/disabled/index.html

```diff
@@ -85,7 +81,6 @@
 
           <ion-item>
             <ion-checkbox aria-label="Checkbox" slot="start"></ion-checkbox>
-            <ion-toggle disabled value="45">Checkbox + Toggle</ion-toggle>
```

Comment:
Could we remove the whole item here instead?

Reply:
I was planning on removing this in my checkbox PR if that's ok? (The checkbox work is blocked by the toggle work)

Reply:
That works 👍